book_source/developers_guide/How-to-insert-new-Input-data.Rmd

```diff
@@ -25,3 +25,15 @@ From your BETY interface:
     + In the Search window, search for the DBFILE you just created
  * Once you have found the DBFILE, click on the “+” icon to add the file
  * Click on “Update” at the bottom when you are done.
+
+
+
+## Create a Format Record 
```

Comment:
Is there not already a page about creating a Format? I'll leave this for now, but in the next revision I'd strongly suggest having a completely separate page for Formats and getting into the details -- right now the information below is missing tons of critical information about how to associate variables with formats and the conventions used there.